### PR TITLE
Added Polkadot Parachains (tada!) to dev: Acala, Moonbeam, Astar, Par…

### PR DESCRIPTION
**Pull request by antonkhvorov**
Added Polkadot Parachains (tada!) to dev: Acala, Moonbeam, Astar, Parallel, Clover

Upd: + Statemint

### CODE REVIEW

**Review by valentunn — APPROVED** [review]
Geneses checked ✅ 

Everything else looks fine too

File: chains/v2/chains_dev.json [hunk 1]
```diff
@@ -351,13 +351,25 @@
...
+                "extrinsic": "https://moonbeam.subscan.io/extrinsic/{hash}",
+                "account": "https://moonbeam.subscan.io/account/{address}",
+                "event": null
             }
         ],
         "types": {
-            "url": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/chains/v2/types/moonriver.json",
-            "overridesCommon": true
+            "url": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/chains/v2/types/moonbeam.json",
+            "overridesCommon": false
```
Comment (ERussel) [inline chains/v2/chains_dev.json:372]: As I see in polkadot.js, runtime is v14. So this field should be true.

File: chains/v2/chains_dev.json [hunk 2]
```diff
@@ -958,5 +970,199 @@
...
+        "assets": [
+            {
+                "assetId": 0,
+                "symbol": "PARA",
+                "precision": 12
+            }
+        ],
+        "nodes": [
+            {
+                "url": "wss://rpc.astar.network",
```
Comment (ERussel) [inline chains/v2/chains_dev.json:1075]: Wrong nodes

File: chains/v2/chains_dev.json [hunk 3]
```diff
@@ -958,5 +970,199 @@
...
+                "name": "Astar node"
+            },
+            {
+                "url": "wss://astar.api.onfinality.io/public-ws",
+                "name": "OnFinality node"
+            }
+        ],
+        "explorers": [
+            {
+                "name": "Subscan",
```
Comment (ERussel) [inline chains/v2/chains_dev.json:1085]: Wrong explorers

**Comment by antonkhvorov** [comment]
Thank you @ERussel @valentunn ✨

**Pull request merged**